packages/components/src/templates/next/components/internal/ChildrenPages/ChildrenPages.tsx

```diff
@@ -156,6 +224,18 @@ const ChildrenPages = ({
     )
   }
 
+  if (layout === "boxes") {
+    return (
+      <BoxLayout
+        assetsBaseUrl={site.assetsBaseUrl}
+        childpages={children}
+        showSummary={summary}
+        showThumbnail={thumbnail}
+        fallback={{ src: site.logoUrl, alt: "Default logo of the site" }}
+      />
+    )
+  }
+
   return (
```

Comment:
question: since layout can only be `boxes` or `rows`, the return statement below for Infocards are purely a temporary catch-all before we migrate everything over?